### PR TITLE
[BAQE-1546] - revapi clean-up process should be automated

### PR DESCRIPTION
**Pull request by afalhambra**
**[BAQE-1546](https://issues.redhat.com/browse/BAQE-1546)**: Revapi clean-up process should be automated

### CODE REVIEW

**Review by mbiarnes — APPROVED** [review]
LGTM

File: Jenkinsfile.revapi [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    }
+
+    parameters {
+        string(name: 'DISPLAY_NAME', defaultValue: '', description: 'Setup a specific build display name')
+
+        // Git information
+        string(name: 'BUILD_BRANCH_NAME', defaultValue: 'master', description: 'Which branch to build? Set if you are not on a multibranch pipeline.')
+        string(name: 'GIT_AUTHOR', defaultValue: 'kiegroup', description: 'Which Git author repository ?')
+        // Revapi branch name
+        string(name: 'REVAPI_RELEASE_VERSION', defaultValue: '', description: 'Release version to apply revapi clean to. E.g. 7.49.Final')
```
Comment (MarianMacik) [inline Jenkinsfile.revapi:33]: Here I would change the description so it says it is actually a version revapi should check against. We are applying cleaning always to master.
Also versions should be with the last 0, e.g. `7.49.0.Final`.

File: Jenkinsfile.revapi [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+String getRevapiProjectName() {
+    return "${getProjectName()}-${getRevapiReleaseVersion()}"
+}
+
+String getRevapiReleaseVersion() {
+    return params.REVAPI_RELEASE_VERSION
+}
+
+void setRepositoriesPropertyIfNeeded(String key, def value){
```
Comment (MarianMacik) [inline Jenkinsfile.revapi:181]: Maybe call it `setRevapiPropertyIfNeeded`?

File: Jenkinsfile.revapi [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    return params.CREATE_PRS
+}
+
+boolean isMergePRs() {
+    return params.MERGE_PRS
+}
+
+boolean isRelease() {
+    return params.RELEASE
+}
```
Comment (MarianMacik) [inline Jenkinsfile.revapi:151]: Not used at all.

**Review by MarianMacik — APPROVED** [review]
Looks good to me. Minor suggestions which are optional.

File: Jenkinsfile.revapi (hunk 3 above)
Comment (afalhambra) [reply Jenkinsfile.revapi]: done

File: Jenkinsfile.revapi (hunk 2 above)
Comment (afalhambra) [reply Jenkinsfile.revapi]: yes, much better

File: Jenkinsfile.revapi (hunk 1 above)
Comment (afalhambra) [reply Jenkinsfile.revapi]: done

**Comment by afalhambra** [comment]
Thanks for your review guys.
@MarianMacik feel free to merge if you're happy with it. Thanks

**Pull request merged**